brainstormforce/ultimate-addons-for-gutenberg
# SPCTR-1269 Fix: Table of Contents - Font decoration not working

### Description
Table Of Contents: Text decoration is not working for all devices on frontend

### Screenshots
Issue - https://d.pr/v/YJKN7L
Fix - https://share.getcloudapp.com/L1uXRpxR

### Types of changes

Bug fix (non-breaking change which fixes an issue)



### How has this been tested?

- Add table of contents block
- Go to list style setting
- In typography set text decoration for all devices 
- Check on frontend

### Checklist:
